tests/sentry/tasks/post_process/tests.py

```diff
@@ -537,6 +539,15 @@ def test_nodestore_stats(self):
 
 
 class PostProcessGroupAssignmentTest(TestCase):
+    def setUp(self):
+        self.cache_keys_to_delete = set()
+
+    def tearDown(self):
+        for key in self.cache_keys_to_delete:
+            cache.delete(key)
```

Comment:
You shouldn't need to do this, since we clear the cache on `_pre_setup`: https://github.com/getsentry/sentry/blob/e403432ccff3d39b2c3bf88cac39ae13912ec4d3/src/sentry/testutils/cases.py#L236

Reply:
Awesome! didn't know that